index.js

```diff
@@ -21,6 +21,16 @@ function renderPlayer(player) {
 
   // append the element to the container
   playerContainer.append(playerDiv)
+
+  const likeBtn = playerDiv.querySelector(".like-button")
+  likeBtn.addEventListener("click", function() {
+    const playerLikes = playerDiv.querySelector(".likes")
+    player.likes += 1 
+    // can't figure out how to keep "likes" when adding a 1 like
+    
+    playerLikes.textContent = player.likes
+  
+  })
```

Comment:
This looks good, nice job using the `player` object to get the likes! To show the string with the number of likes, you can just do something like this:

```js
playerLikes.textContent = `${player.likes} likes`
```